10LessonTen/Checkpoint2/index.html

```diff
@@ -2,9 +2,100 @@
 <html>
 <head>
   <title></title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.min.css">
+  <link href="https://fonts.googleapis.com/css?family=Amatica+SC|Monoton|Shadows+Into+Light+Two" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Cinzel|Poiret+One" rel="stylesheet">
+  <link rel="stylesheet" href="../../css-reset.css"/>
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
-
+  <nav>
+    <span class="left business-name"><a href="#">SIP COFFEE</a></span>
+    <ul class="right">
+      <li><a href="#menu">Menu</a></li>
+      <li><a href="#events">Events</a></li>
+      <li><a href="#bottom">Contact</a></li>
+    </ul>
+  </nav>
+  <header id="top">
+    <div class="overlay">
+    <h1>Sip Coffee</h1>
+    <hr>
+    <p class="description">Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+    <hr>
+  </div>
+  </header>
+  <main>
+    <section id="menu">
+    <h2>Our Offerings</h2>
+    <div class="menuitems">
+    <article class="drink">
```

Comment:
Make sure elements are properly indented; makes it MUCH easier to read (both for you and me!). Atom will do it automatically too!